README.md

```diff
@@ -16,6 +16,17 @@ The [FMI Layered Standard for XCP (fmi-ls-xcp)][spec] is currently maintained on
 [GitHub][githubspec] and is published [here][spec]. It is based on
 the [FMI][] standard.
 
+## Repository Structure
+- `docs` -- Sources of the specification document
+- `schema` -- XSD schema for this FMI Layered
+
+## Copyright and License
+Code and documentation copyright (C) 2008-2011 the MODELISAR consortium and 2012-2024 the Modelica Association Project FMI.
```

Comment:
I think we can leave out MODELISAR here and change the copyright to  2023-2024 as in the ls-xcp spec

Reply:
@chrbertsch good point; thanks! I fixed it and updated the PR.